diff --git a/js/ui/polkitAuthenticationAgent.js b/js/ui/polkitAuthenticationAgent.js
index 9135790..b9fa41f 100644
--- a/js/ui/polkitAuthenticationAgent.js
+++ b/js/ui/polkitAuthenticationAgent.js
@@ -556,7 +556,7 @@ var AuthenticationAgent = class {
             const session = new GnomeSession.SessionManager();
             const csSessionId = session.SessionId;
 
-            let procSessionId = this._native.register();
+            let procSessionId = null; //this._native.register();
             if (procSessionId !== csSessionId) {
                 global.logWarning("Unable to register as the polkit agent for login session '%s'. Cinnamon is running under '%s'"
                                   .format(csSessionId, procSessionId));
